content/en/docs/setup/upgrade/index.md

```diff
@@ -60,6 +73,16 @@ For example, the following command will show all the pods using the `canary` rev
 $ kubectl get pods -n test-ns -l istio.io/rev=canary
 {{< /text >}}
 
+To verify that the new pods in the `test-ns` namespace are using the `istiod-canary` service corresponding to the `canary` revision, select one newly created pod and use the `pod_name` in the following command:
+
+{{< text bash >}}
+$ kubectl  exec -i -t -n test-ns ${pod_name} -c istio-proxy -- curl http://localhost:15000/clusters | grep xds | grep hostname
```

Comment:
```
$ ik pc bootstrap POD | grep discoveryAddress
                                "discoveryAddress": "istiod.istio-system.svc:15012",
```

also works

or 

```
$ ik pc endpoints `kp alpine` --cluster xds-grpc -ojson | grep hostname
                "hostname": "istiod.istio-system.svc",
```

same concept, just avoiding complicated exec commands

Reply:
Changed to use `istioctl proxy-config` instead of kubectl.